Math.md

```diff
@@ -0,0 +1,96 @@
+##Float and Double
+You've already learned about the String and Int types; Strings are values like "hello" or "eat your vegetables", and Ints are whole number values like 34 and 9561.  
```

Comment:
This is a perfect introduction.

I would add something like the following below this initial paragraph so they see what a `String` looks like again.

```swift
let greeting = "hello"
let momsOrder = "eat your vegetables"
```